packages/packages/api/src/medical/client/metriport.ts

```diff
@@ -56,4 +59,57 @@ export class MetriportMedicalApi {
     if (!resp.data) return undefined;
     return resp.data;
   }
+
+  /**
+   * Builds and returns the current state of a patient's links across HIEs.
+   *
+   * @param {string}    patientId    Patient ID for which to retrieve links.
+   *
+   * @returns {PatientLinks}         The patient's current and potential links.
+   */
+  async getLinks(patientId: string): Promise<PatientLinks> {
+    const resp = await this.api.get<PatientLinks>(
+      `${this.PATIENT_URL}/${patientId}${this.LINK_URL}`
+    );
+
+    if (!resp.data) throw new Error(`Get didn't return Links`);
+    return resp.data;
+  }
+
+  /**
+   * Creates link to the specified entity.
+   *
+   * @param {string}                     patientId    Patient ID for which to retrieve links.
+   * @param {string}                     entityId     Entity ID to link to the patient.
+   * @param {ExternalMedicalPartners}    linkSource   HIE from where the link is made too..
```

Comment:
nit: `Data source to link to.`

Not all of the data sources will be HIEs in the future.